src/NtpClient.ycp

```diff
@@ -281,6 +281,8 @@ global list GetNtpServersByCountry ( string country, boolean terse_output ) {
         country_names = GetCountryNames ();
     }
 
+    boolean default_already_chosen = false;
+    boolean selected = false;
```

Comment:
NP: You do not need to keep the `selected` value between the iterations or after the `maplist` call, it would be better to move it inside the iteration to have a smaller scope.